solutions/src/main/java/ru/sberbank/school/task11/Streams.java

```diff
@@ -0,0 +1,124 @@
+     *
+     * @param elements входные элементы
+     * @return стрим элементов из elements
+     */
+    public static Streams of(Object... elements) {
+        return null;
+    }
+
+    /**
+     * Принимает на вход коллекцию элементов и возвращает стрим построенный на основе этих элементов.
+     *
+     * @param elements входные элементы
+     * @return стрим элементов из elements
+     */
+    public static Streams of(Collection elements) {
+        return null;
+    }
+
+    /**
+     * Фильтрация элементов по заданному правилу. Необходимо подобрать нужый интерфейс для передачи в этот метод.
+     * <p>
+     * Иными словами, этот метод оставляет в коллекции только те элементы, которые удовлетворяют условию в лямбде.
+     * <p>
+     * Аналог Stream#map(Function).
+     * <p>
+     * Intermediate операция.
+     *
+     * @param object - правило фильтрации элементов.
+     * @return стрим
+     */
+    public Streams filter(Object object) {
+        return null;
+    }
+
+    /**
+     * Преобразование элементов в какие-то другие элементы. Необходимо подобрать нужый интерфейс для передачи в этот метод.
+     * <p>
+     * Например, данный метод должен уметь извлекать какие-то поля из объектов исходного типа (если это возможно).
+     * <p>
+     * Аналог Stream#map.
+     * <p>
+     * Intermediate операция.
+     *
+     * @param object - правило траснформации элементов.
+     * @return стрим
+     */
+    public Streams transform(Object object) {
+        return null;
+    }
+
+    /**
+     * Сортировка элементов. Необходимо подобрать нужый интерфейс для передачи в этот метод.
+     * <p>
+     * Данный метод должен уметь сортировать элементы по заданному правилу.
+     * <p>
+     * Аналог Stream#sorted
+     * <p>
+     * Intermediate операция.
+     *
+     * @param object - правило сортировки элементов.
+     * @return стрим
+     */
+    public Streams sorted(Object object) {
+        return null;
+    }
+
+    /**
+     * Преобразование стрима в Map. Необходимо подобрать нужные интерфейсы для передачи в этот метод.
+     * <p>
+     * Данный метод должен уметь преобразовать стрим в Map используя keyMapper и valueMapper.
+     * <p>
+     * Terminate операция.
+     *
+     * @param keyMapper   - правило создания ключа.
+     * @param valueMapper - правило создания значения.
+     * @return Map, собранная по правилам keyMapper и valueMapper
+     */
+    public Map toMap(Object keyMapper, Object valueMapper) {
+        return null;
+    }
+
+    /**
+     * Преобразование стрима в Set.
+     * <p>
+     * Данный метод должен уметь преобразовать стрим в Set. Обратите внимание, что метод sorted должен учитываться
+     * при вызове этого метода.
+     * <p>
+     * Terminate операция.
+     *
+     * @return Set элементов
+     */
+    public Set toSet() {
+        return null;
+    }
+
+    /**
+     * Преобразование стрима в List.
+     * <p>
+     * Данный метод должен уметь преобразовать стрим в List.
+     * <p>
```

Comment:
&lt;p&gt; tag should be placed immediately before the first word, with no space after.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-2076734968>*